### PR TITLE
745 generate context typescript files for 2.0 (and apply prettier)

### LINKED ISSUE #745: Generate typescript types for contexts added in 2.0
Prior to release the FDC3 source and NPM module need to be updated with new typescript types for standard contexts generated from the schemas. These go in [src/context/ContextType.ts](https://github.com/finos/FDC3/blob/master/src/context/ContextType.ts) and [src/context/ContextTypes.ts](https://github.com/finos/FDC3/blob/master/src/context/ContextTypes.ts).

A script is provided to do this: https://github.com/finos/FDC3/blob/544-Desktop-Agent-Bridging-Proposal/package.json/#L30-L31
`yarn typegen`

### PR DESCRIPTION
**Pull request by kriswest**
resolves #745

### CODE REVIEW

File: src/context/ContextTypes.ts [hunk 1]
```diff
@@ -40,48 +103,61 @@ export interface ContactID {
...
+  subject?: string;
+  textBody?: string;
   type: string;
   id?: { [key: string]: string };
   name?: string;
 }
 
-export interface Instrument {
-  id: InstrumentID;
+export interface RecipientsObject {
```
Comment (rikoe) [inline src/context/ContextTypes.ts:138]: I don't like `RecipientsObject` as a name for developers to use. Names should be domain-based, not with development concepts like "object" in them.

This is probably a `QuickType` thing, and can hopefully be tweaked by playing with the command line parameters used in package.json.

If not, I think we should investigate these options:
* Try to control the name QuickType generates through its settings
* Manually change the file after generation
* Automatically change the file after generation

**Review by rikoe — CHANGES_REQUESTED** [review]

File: src/context/ContextTypes.ts (hunk 1 above)
Comment (kriswest) [reply src/context/ContextTypes.ts:138]: I'm not hugely bothered by this as I doubt anyone will use that generated type directly. Its only exported to facilitate type checking. 

I'm also not seeing an option to fix it in the [TypeScript flow renderer in quicktype](https://github.com/quicktype/quicktype/blob/master/src/quicktype-core/language/TypeScriptFlow.ts) (the javascript name option doesn't do it. You can try it here: https://app.quicktype.io/

We may, however, wish to disable the javascript name option as it is replacing capitalized properties or those with underscores with camelcase... Pretty sure we don't want that.

Comment (kriswest) [reply src/context/ContextTypes.ts:138]: Seems like losing the `*Object` names would require at least a custom language definition - which didn't defer to the code generating the names in the ConvenienceRenderer: https://blog.quicktype.io/customizing-quicktype/

Comment (rikoe) [reply src/context/ContextTypes.ts:138]: Okay, I agree it's not a biggie, we can always add a separate issue, and release an update of the npm package only if we find a way to deal with it. I will approve.

**Review by rikoe — APPROVED** [review]

**Review by mistryvinay — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**